app/models/mixins/ui_service_mixin.rb

```diff
@@ -1,21 +1,24 @@
 module UiServiceMixin
+  # rubocop:disable Metrics/LineLength
```

Comment:
There are several levels of rubocop errors and warnings. Some are "must" (red), the others are really nice to be fixed. "Too long lines" are not always fixed.
Generally speaking, I'm not in favor of "mixing in" non source code related things such as style checks remarks into the source code. This is an external check that runs on the code.
Also, consider this - today it's rubocop, and tomorrow it might be replaced by another solution.
Putting the rubocop comments in code will require a huge cleanup.